simphony/cuds/abc_modeling_engine.py

```diff
@@ -37,16 +37,9 @@ def add_lattice(self, lattice):
 
         Parameters
         ----------
-        lattice: ABCLattice
+        lattice : ABCLattice
             lattice to be added.
 
-        Returns
```

Comment:
>Having the add_XXX methods return a proxy object could confuse since in-general you would not expect that an add method would return a similar object back.
yeah, i can see that could be confusing.  yeah, issue to discuss this sounds good!